hbase-hadoop2-compat/src/main/java/org/apache/hadoop/hbase/ipc/MetricsHBaseServerSourceImpl.java

```diff
@@ -41,7 +40,8 @@
   private final MutableFastCounter authenticationFallbacks;
   private final MutableFastCounter sentBytes;
   private final MutableFastCounter receivedBytes;
-
+  private final MutableFastCounter numCallInResponseQueue;
```

Comment:
I think MutableGaugeLong is the right type for this.. Counters are meant to be always increasing where as a guage can go up and down...